fcontext fails to configure on Stria (arm64)

```
CMake Error at TPL/vt/lib/context/CMakeLists.txt:77 (add_library):
  Cannot find source file:

    /lustre/dcollin/empire/src/EMPIRE/TPL/vt/lib/context/src/context/asm/make_aarch64_sysv_elf_gas.S

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .mpp .m .M .mm .h .hh .h++
  .hm .hpp .hxx .in .txx .f .F .for .f77 .f90 .f95 .f03 .ispc


CMake Error at TPL/vt/lib/context/CMakeLists.txt:77 (add_library):
  No SOURCES given to target: fcontext
  ```

--- a/lib/context/CMakeLists.txt
+++ b/lib/context/CMakeLists.txt
@@ -24,6 +24,10 @@ install(
 EXECUTE_PROCESS(COMMAND uname -p OUTPUT_VARIABLE CPU_ARCH )
 string(REGEX REPLACE "\n$" ""  CPU_ARCH "${CPU_ARCH}" )
 
+if (CPU_ARCH MATCHES "aarch64")
+  set(CPU_ARCH "arm64")
+endif()
+
 if (CPU_ARCH MATCHES "ppc")
   set(CPU_ARCH "ppc64")
 endif()

